lib/howl/ui/editor.moon

```diff
@@ -367,16 +367,34 @@ class Editor extends PropertyObject
       return
 
     conf = @config_at_cursor
-    if conf.tab_indents and @current_context.prefix.is_blank
-      cur_line = @current_line
-      next_indent = cur_line.indentation + conf.indent
-      next_indent -= (next_indent % conf.indent)
-      cur_line.indentation = next_indent
-      @cursor.column = next_indent + 1
-    else if conf.use_tabs
+
+    if conf.use_tabs
       @view\insert '\t'
-    else
+      return
+
+    -- !NOTE!
```

Comment:
`tab_indents = false` is most noticeable when `use_tabs` is set to true I think. Consider:

```moonscript
foo ->
  bar!
```
With the cursor at the start of the second line and use tabs set to true, try toggling the option back and forth and see the difference.

WRT `indent`, I think that `indent` is widely used when referring to coding standards, whereas tabstop-width does not ring a bell. I don't think that sounds like a good change personally.